telemetry/definitions/commonDefinitions.json

```diff
@@ -129,7 +129,7 @@
             "name": "credentialSourceId",
             "type": "string",
             "description": "Where credentials are stored or retrieved from",
-            "allowedValues": ["sharedCredentials", "sdkStore", "ec2", "envVars", "other"]
+            "allowedValues": ["sharedCredentials", "sdkStore", "ec2", "container", "envVars", "other"]
```

Comment:
alternatively, "ecs", but the java sdk refers to this generically

Reply:
would prefer "ecs" for parallel form with the existing "ec2"